.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: ['10', '12', '14']
+        node: ['10', '12', '14', '16']
```

Comment:
Adding node 16, which is getting common. Node 10 is EOL and could be removed in the future (AWS dropped support recently)